web/themes/custom/fmcg_material/sass/_button.scss

```diff
@@ -55,20 +55,30 @@ input.button, .btn, a.btn, span.btn, .tabledrag-toggle-weight, .button-action, .
   }
 }
 
-.js .dropbutton-widget .dropbutton {
+.dropbutton-widget, .dropbutton {
   overflow: hidden;
   margin: -12px;
   padding: 0;
   list-style-type: none;
   list-style-image: none;
+
+  .secondary-action {
+    margin: -8px 0px 0px 0px;
+  }
 }
 
-li.approve, li.reject {
+.dropbutton a {
   background-color: $fmcg-chrome;
   border-top-left-radius: 2px;
   border-bottom-left-radius: 2px;
   padding: 8px 12px;
 
+  &:hover {
+    background-color: $fmcg-black;
+    color: $fmcg-chrome;
+    border: 2px solid $fmcg-chrome;
+  }
+
   a {
```

Comment:
So this is now going to apply to any a tag inside an a tag inside something with the class drop button.

I think that means its not actually going to apply to anything, so we can probably remove

```html
<div class="dropbutton">
  <a>
    <a></a><!-- Will only apply to this internal a tag -->
  </a>
</div>
```

Reply:
Ah I understand this better now, this diagram helps me to better see the hierarchy.